components/Tabs/Tabs.js

```diff
@@ -1,24 +1,27 @@
-
 class TabItem {
   constructor(element) {
     // attach dom element to object. Example in Tabs class
+    this.element = element;
   }
 
   select() {
     // should use classList
+    const item = document.querySelector('.List .List__item'); // `.querySelector` selects the class `.List` wich holds another class, `.List__item` which is the class attached to the 3 buttons (tabs) that we want. We are attaching this result to the `const item` so we can easily use it in the line below.
+    this.item.select();
```

Comment:
Line 10 doesn't do anything other than add a query selector.  It doesn't perform any action, and the selection is only valid within this function.